account_financial_report/report/general_ledger.py

```diff
@@ -335,14 +336,8 @@ def _get_move_line_data(self, move_line):
             "rec_name": move_line["full_reconcile_id"][1]
             if move_line["full_reconcile_id"]
             else "",
-            "tag_ids": move_line["analytic_tag_ids"],
```

Comment:
Why no tags now?

Reply:
account.analytic.tag has been removed in v16